config/webpack.common.js

```diff
@@ -288,11 +280,33 @@ const addLoaders = ( conf ) => {
 		},
 	];
 
+	// When importing a module whose path matches one of the following, just
+	// assume a corresponding global variable exists and use that instead.
+	// This is important because it allows us to avoid bundling all of our
+	// dependencies, which allows browsers to cache those libraries between builds.
+	enhancedConf.externals = {
+		react: 'React',
+		'react-dom': 'ReactDOM',
+	};
+
```

Comment:
This config is from this docs:
https://www.typescriptlang.org/docs/handbook/react-&-webpack.html
![image](https://user-images.githubusercontent.com/1477453/71446047-f1abd700-2727-11ea-9123-d3353931094d.png)